compat/poll/poll.c

```diff
@@ -462,7 +462,7 @@ poll (struct pollfd *pfd, nfds_t nfd, int timeout)
   if (timeout != INFTIM)
     {
       orig_timeout = timeout;
-      start = GetTickCount();
+      start = GetTickCount64();
```

Comment:
`start` and`elapsed` are `DWORD`, while `GetTickCount64` returns `ULONGLONG`. Wouldn't that be a problem?